src/app/App.tsx

```diff
@@ -22,17 +28,21 @@ const App: FC<AppProps> = ({ env }) => (
   <ErrorBoundary whileMessage="booting a wallet" className="min-h-screen">
     <DialogsProvider>
       <Suspense fallback={<RootSuspenseFallback />}>
-        <AppProvider env={env}>
-          <Dialogs />
+        <Provider store={store}>
```

Comment:
mb move this to ConditionalReadyTemple?

Reply:
no

Reply:
because redux is used inside pageRouter, which is using WoozieProvider. Thus, the Woozie provider itself must be wrapped with redux context - either in app provider or even above this parent